alacritty_terminal/src/term/cell.rs

```diff
@@ -184,11 +211,22 @@ impl LineLength for grid::Row<Cell> {
 
 #[cfg(test)]
 mod tests {
+    use std::mem;
+
     use super::{Cell, LineLength};
```

Comment:
```suggestion
    use super::*;

    use std::mem;
```

I think this would be more consistent with the rest of our code.